ts/Shared/Utilities.ts

```diff
@@ -16,22 +16,21 @@
  *
  * */
 
-import type AxisType from './Axis/AxisType';
-import type Chart from './Chart/Chart';
-import type CSSObject from './Renderer/CSSObject';
+import type AxisType from '../Core/Axis/AxisType';
+import type Chart from '../Core/Chart/Chart';
+import type CSSObject from '../Core/Renderer/CSSObject';
 import type {
     DOMElementType,
     HTMLDOMElement
-} from './Renderer/DOMElementType';
-import type EventCallback from './EventCallback';
-import type HTMLAttributes from './Renderer/HTML/HTMLAttributes';
-import type Series from './Series/Series';
-import type SVGAttributes from './Renderer/SVG/SVGAttributes';
-import type Time from './Time';
-
-import H from './Globals.js';
+} from '../Core/Renderer/DOMElementType';
+import type EventCallback from '../Core/EventCallback';
+import type HTMLAttributes from '../Core/Renderer/HTML/HTMLAttributes';
+import type Series from '../Core/Series/Series';
+import type SVGAttributes from '../Core/Renderer/SVG/SVGAttributes';
+import type Time from '../Core/Time';
+
+import H from '../Core/Globals.js';
```

Comment:
It does not help to just move the file, if the dependencies are kept. `Shared`-files should not have imports at all, and if really necessary these should be in `Shared` as well. The best `Shared`-files are atomic (self-containing) ones.